contracts/ReversibleICO.sol

```diff
@@ -260,7 +260,7 @@ contract ReversibleICO is IERC777Recipient {
     )
     public view returns (uint256)
     {
-        // return (_token_amount / StageByNumber[_stageId].token_price) / (10 ** 18));
+        // return (_token_amount * StageByNumber[_stageId].token_price) / (10 ** 18);
```

Comment:
What does the 11 ** 18 do?
The token price should be in WEI, rather than ETH. Or is it already?